src/App.tsx

```diff
@@ -1,7 +1,33 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends Component {
+  state = {
+    lastKey: '',
```

Comment:
```suggestion
    pressedKey: '',
```
![image](https://user-images.githubusercontent.com/96409563/215697650-0520aefa-37ea-41eb-ba8d-422f378b78e8.png)